libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/accounts/UserAccountTest.java

```diff
@@ -247,57 +381,213 @@ private Bundle createTestAccountBundle() {
         object.putString(UserAccount.LOCALE, TEST_LOCALE);
         object.putString(UserAccount.PHOTO_URL, TEST_PHOTO_URL);
         object.putString(UserAccount.THUMBNAIL_URL, TEST_THUMBNAIL_URL);
-        object.putString(UserAccount.LIGHTNING_DOMAIN, null);
-        object.putString(UserAccount.LIGHTNING_SID, null);
-        object.putString(UserAccount.VF_DOMAIN, null);
-        object.putString(UserAccount.VF_SID, null);
-        object.putString(UserAccount.CONTENT_DOMAIN, null);
-        object.putString(UserAccount.CONTENT_SID, null);
-        object.putString(UserAccount.CSRF_TOKEN, null);
-        object.putBoolean(UserAccount.NATIVE_LOGIN, false);
+        object.putString(UserAccount.LIGHTNING_DOMAIN, TEST_LIGHTNING_DOMAIN);
+        object.putString(UserAccount.LIGHTNING_SID, TEST_LIGHTNING_SID);
+        object.putString(UserAccount.VF_DOMAIN, TEST_VF_DOMAIN);
+        object.putString(UserAccount.VF_SID, TEST_VF_SID);
+        object.putString(UserAccount.CONTENT_DOMAIN, TEST_CONTENT_DOMAIN);
+        object.putString(UserAccount.CONTENT_SID, TEST_CONTENT_SID);
+        object.putString(UserAccount.CSRF_TOKEN, TEST_CSRF_TOKEN);
+        object.putBoolean(UserAccount.NATIVE_LOGIN, TEST_NATIVE_LOGIN);
+        object.putString(UserAccount.COOKIE_CLIENT_SRC, TEST_COOKIE_CLIENT_SRC);
+        object.putString(UserAccount.COOKIE_SID_CLIENT, TEST_COOKIE_SID_CLIENT);
+        object.putString(UserAccount.SID_COOKIE_NAME, TEST_SID_COOKIE_NAME);
+        object.putString(UserAccount.CLIENT_ID, TEST_CLIENT_ID);
         object = MapUtil.addMapToBundle(createAdditionalOauthValues(), createAdditionalOauthKeys(), object);
         return object;
     }
 
     /**
-     * Check for equality of two bundles.
-     *
-     * @param one the first bundle
-     * @param two the second bundle
-     * @return true if the keys/values match
-     *         false otherwise
+     * Create test account
+     */
+    public static UserAccount createTestAccount() {
```

Comment:
Static method to create a test account (`UserAccount` object not the account manager account) that this test suite and others all use (e.g. `UserAccountManagerTest` and `ClientManagerTest`).

Also creating the account manager account is a lot simpler now. 
No need to set up `LoginOptions` / `ClientManager` and call a method that takes many args, simply call `UserAccountManager.createAccount(userAccount)`.